style/components/snackbar.css

```diff
@@ -0,0 +1,40 @@
+#snackbar {
```

Comment:
Use classname instead of id, you may need this snack bar in other places also it will be an issue then.